puppet/zulip/manifests/nginx.pp

```diff
@@ -58,6 +58,11 @@
     source  => 'puppet:///modules/zulip/nginx/dhparam.pem',
   }
 
+  if $::osfamily == 'debian' {
+      $ca_crt = 'ca-certificates.crt'
+  } else {
+      $ca_crt = 'ca-bundle.crt'
+  }
```

Comment:
We should put the full path in this variable, and use `/etc/pki/tls/certs/ca-bundle.crt` on CentOS ([explanation](https://www.happyassassin.net/2015/01/12/a-note-about-ssltls-trusted-certificate-stores-and-platforms/)).

Reply:
Fixed.